ocs_ci/utility/deployment_openshift_logging.py

```diff
@@ -311,3 +313,61 @@ def create_instance():
     # Get the CSV installed
     get_csv = csv_obj.get(out_yaml_format=False)
     logger.info(f"The installed CSV is {get_csv}")
+
+
+def install_logging():
```

Comment:
@Shrivaibavi Do review this function, can't we re-use any existing logging functions or fixtures to deploy logging?

Reply:
Yes, currently logging installation is part of fixture. I would need to start logging in threads so created a new logging installation function. Once this code is merged, will replace the existing code in the fixture with this function call after thorough testing.

Reply:
@ramkiperiy We have a fixture install_logging in /tests/conftest.py, 
@PrasadDesala Lines 318-373 can be removed and replaced with above mentioned fixture. 
def install_logging_on_longetivity_cluster(install_logging):
    logging.info(".........")